distro/pom.xml

```diff
@@ -126,6 +126,7 @@
               <outputDirectory>${project.build.directory}/openmrs_modules/</outputDirectory>
               <!-- copying jars and omods only -->
```

Comment:
Let's rename to:
> Copying JARs and OMODs only.
Exclude any 'com.ozonehis' artifact to make sure EIP routes are not copied over